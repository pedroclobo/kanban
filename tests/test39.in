t 12 análise de requisitos
t 8 desenho
t 34 realização
t 24 teste
t 17 instalação
d TO DO
q
