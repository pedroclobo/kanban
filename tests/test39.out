task 1
task 2
task 3
task 4
task 5
1 0 análise de requisitos
2 0 desenho
5 0 instalação
3 0 realização
4 0 teste
